src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-time;
```

Comment:
list all transition properties, it is better for performance

Reply:
checklist!
![image](https://user-images.githubusercontent.com/98764641/210969845-565c817e-4ace-4f9c-a1b1-79e2a17c2f78.png)

Reply:
> checklist! ![image](https://user-images.githubusercontent.com/98764641/210969845-565c817e-4ace-4f9c-a1b1-79e2a17c2f78.png)

Please specify what you mean. 'transition' was added not to ':hover', it was added to the element which I want to smoothly transform and change color.